array-destructuring/exercise-2/exercise.js

```diff
@@ -70,3 +70,11 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+
+let GryfArray = hogwarts.filter((student) => {
+  return (student.house === "Gryffindor")
+})
```

Comment:
Do you need to wrap the condition in brackets?

Is there anything that is missing on these two lines?